apps/barnepensjon-ui/src/utils/age.ts

```diff
@@ -13,7 +13,12 @@ export const getAgeFromDate = (date: Date | string): number => {
 
 // Støtter også D-nummer.
 export const getAgeFromFoedselsnummer = (foedselsnummer: string): number => {
-    const birthDate = navFaker.personIdentifikator.getFødselsdato(foedselsnummer)
+    try {
+        const birthDate = navFaker.personIdentifikator.getFødselsdato(foedselsnummer)
 
-    return getAgeFromDate(birthDate)
+        return getAgeFromDate(birthDate)
+    } catch (e) {
```

Comment:
Vi har ikke behov for den reelle alderen? 
Kan det være et problem hvis dette skjer for andre identifikatorer?
Det er ingen måte å oppdage at det er testfnr?

Reply:
Generelt bør vi ikke avlede fødselsdato fra fnr.

Reply:
Høres ut som noe man burde se på da, men denne PR:en er ju uansett greit å få in. Den fikser ju en error i systemet uten å endre noe funksjonalitet, det finnes ju allerede inne.